src/sentry/migrations/0333_savesearch_add_visibility.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.28 on 2022-10-27 04:53
+
+from django.db import migrations
+import sentry.db.models.fields.text
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not. Generally
+    # we don't want to run in a transaction here, since for long running operations like data
+    # back-fills this results in us locking an increasing number of rows until we finally commit.
+    atomic = False
+
+
+    dependencies = [
+        ('sentry', '0332_release_user_agent'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='savedsearch',
+            name='visibility',
+            field=sentry.db.models.fields.text.CharField(default='owner', max_length=16, null=True),
```

Comment:
A few things

I've set the default for visibility to "owner only".

@wedamija - When this migration runs is it going to leave them all as `NULL` since it's `null=True` right now, or will it set them all to `owner` since that's the default?

Right now I believe saved searches are ALWAYS organization wide, so maybe we actually want to backfill all the existing saved searches with the `org` visibility. In which case, it would be preferable for this migration to create the column as `NULL`.